mobile/src/components/BottomNavBar.tsx

```diff
@@ -1,20 +1,33 @@
 import { Pressable, View } from "react-native";
 import Ionicons from "@expo/vector-icons/Ionicons";
+import { styled } from 'nativewind';
+
+const StyledView = styled(View);
```

Comment:
I don't think that we actually need to use `styled()` on the built in components, the main use is to get the correct typescript typings for our own components.